test/publishing_api_v2_test.rb

```diff
@@ -260,7 +259,7 @@ def content_item_for_content_id(content_id, attrs = {})
       it "responds with 200 and the content item" do
         response = @api_client.get_content(@content_id)
         assert_equal 200, response.code
-        assert_equal @content_item["format"], response["document_type"]
+        assert_equal @content_item["schema_name"], response["document_type"]
```

Comment:
This should probably assert that `assert_equal @content_item["document_type"], response["document_type"]`